pysam/libcalignmentfile.pyi

```diff
@@ -201,7 +201,7 @@ class IteratorRowSelection(IteratorRow): ...
 
 class IteratorColumn:
     def __iter__(self) -> IteratorRow: ...
```

Comment:
While you are here anyway, can you fix this to `IteratorColumn` as @jmarshall noted in https://github.com/pysam-developers/pysam/issues/1129#issuecomment-1264739672?
(I added these type annotations originally and can confirm that this was just an oversight. In this case probably a copy&paste error.)